README.md

```diff
@@ -4,50 +4,18 @@
 Central storage, management and access for important geospatial datasets
 Developed by [Land Information New Zealand](https://github.com/linz)
 
+## Development setup
 
-## Dependencies Installation
-### Python Virtual Environment (for Python CLI and AWS CDK)
-* Create and activate a Python virtual environment
+One-time setup:
 
-    ```bash
-    $ python3 -m venv .venv
-    $ source .venv/bin/activate
-    ```
-* Upgrade pip
-
-    ```bash
-    $ pip install --upgrade pip
-    ```
-* [Install Poetry](https://python-poetry.org/docs/#installation)
-* Install the dependencies:
-
-    ```bash
-    $ poetry install --extras='cdk datasets-endpoint'
-    ```
+1. [Install and enable `pyenv`](https://github.com/pyenv/pyenv#installation).
+   - Verify by running `diff <(python <<< 'import platform; print(platform.python_version())') .python-version`.
+1. [Install Poetry](https://python-poetry.org/docs/#installation).
+1. [Install and enable `nvm`](https://github.com/nvm-sh/nvm#installing-and-updating).
+1. [Install latest `npm` LTS](https://github.com/nvm-sh/nvm#long-term-support).
+1. Run `./reset-dev-env.bash` to install packages.
 
-### AWS CDK Environment (AWS Infrastructure)
```

Comment:
I would prefer keeping those copy/paste commands to having URLs to installation  instructions. If you want to keep those URLs it is OK, but just as additional information to copy/paste commands.

Reply:
Also it would be great to have Python build deps installation instructions

```
$ sudo apt-get update; sudo apt-get install --no-install-recommends make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
```

Reply:
This looks like as a nice instructions https://realpython.com/intro-to-pyenv/#why-use-pyenv

Reply:
Having *both* the links and the instructions is a good compromise. I'll do that. Thanks!

Reply:
Thanks

Reply:
I tested process and works ! I'll have few small comments in new thread.